src/main/java/net/dv8tion/jda/core/hooks/ListenerAdapter.java

```diff
@@ -651,13 +650,14 @@ else if (event instanceof GenericVoiceChannelEvent)
             onGenericVoiceChannel((GenericVoiceChannelEvent) event);
         else if (event instanceof GenericCategoryEvent)
             onGenericCategory((GenericCategoryEvent) event);
-        else if (event instanceof GenericGuildEvent)
-            onGenericGuild((GenericGuildEvent) event);
         else if (event instanceof GenericRoleEvent)
             onGenericRole((GenericRoleEvent) event);
         else if (event instanceof GenericEmoteEvent)
             onGenericEmote((GenericEmoteEvent) event);
 
+        if (event instanceof GenericGuildEvent)
```

Comment:
put a comment here specifying why this is by itself (generic events that have 2 levels of generic subclasses)